conan/tools/gnu/autotools.py

```diff
@@ -21,6 +22,24 @@ def configure(self, build_script_folder=None):
         http://jingfenghanmax.blogspot.com.es/2010/09/configure-with-host-target-and-build.html
         https://gcc.gnu.org/onlinedocs/gccint/Configure-Terms.html
         """
+        configure_args = []
```

Comment:
I had to move setting the --prefix, etc... here so that the local flow works...
The build helper will only set the arguments related with the output folders... and the other arguments come from the toolchain.

Reply:
Not sure about this, lets see what @lasote thinks

Reply:
I would need to know why the local flow is not working. If you want to call the build system directly, you have to look at the `conanbuild.conf` to see which arguments to apply according to the calculated by the toolchain, right? But the `conan build` would work in any case. I'm missing something.